spec.html

```diff
@@ -22543,6 +22550,24 @@ <h1>ExecuteModule ( ) Concrete Method</h1>
             1. Return Completion(_result_).
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-source-text-module-record-get-requested-modules" aoid="GetRequestedModules">
+          <h1>GetRequestedModules ( ) Concrete Method</h1>
+
+          <p>The GetRequestedModules concrete method of a Source Text Module Record implements the corresponding Cyclic Module Record abstract method.</p>
+
+          <p>This abstract method performs the following steps:</p>
+
+          <emu-alg>
+            1. Let _module_ be this Source Text Module Record.
+            1. Let _requestedModules_ be an empty List of Abstract Module Records.
+            1. For each String _requested_ that is an element of _module_.[[RequestedModules]], do
+              1. Let _requestedModule_ be ! HostResolveImportedModule(_module_, _requested_).
```

Comment:
The specification previously had `?` rather than `!` from some paths, so I think it would be best to leave it that way here. Then, the two callsites can invoke it with either of those, based on whether they anticipate errors.

Reply:
Thanks!  I like the idea of moving [[RequiredModules]] down to STMR...I'll look into this.

Reply:
I pushed an update that:
- Switched '?' to '!' in GetRequestedModules and removed the note stating that the HostResolveImportedModules call must have succeeded.
- Pushed [[RequestedModules]] down to Source Text Module Record.

Thanks for the comments!